.github/workflows/go.yml

```diff
@@ -82,6 +82,17 @@ jobs:
     - name: Test registry server with PostgreSQL
       run: go test ./server/registry -postgresql
 
+    - name: Verify that benchmarks run
+      env:
+        APG_REGISTRY_ADDRESS: localhost:8080 
+        APG_REGISTRY_AUDIENCES: http://localhost:8080
+        APG_REGISTRY_INSECURE: 1
+        PROJECTID: bench
+        ITERATIONS: 1
+      run: |
+        registry rpc admin create-project --project_id=$PROJECTID --json
```

Comment:
This works because a server is started above - I believe the one on line 65 is still running. I think @theganyo might remove that line soon, and this will break as a surprise bonus which we can either fix by starting a server here or starting an embedded server using new tricks.

Reply:
It wasn't my intention to remove that as I consider this to be part of the integration test suite.

Reply:
@theganyo OK, leaving that in is fine with me (as is taking it out). I'm not that interested in regularly benchmarking against SQLite since all our deployments will use Postgres. I added this to CI only to ensure that checkins don't break the benchmarking code.